### PR TITLE
feat(icon-button-toggle): create new web component icon-button-toggle

### PR DESCRIPTION
**Pull request by bsahitya**
## Description

Created a new web component called `cv-icon-button-toggle`.
It extends the functionality from [`mwc-icon-button-toggle`](https://www.npmjs.com/package/@material/mwc-icon-button-toggle/v/0.24.0-canary.8dff8894.0)

### What's included?

Icon button toggle component which can be toggled on/off and disabled.
It has the below properties:
- aria-label: Accessible label for the button.
- on: To toggle the button on/off.
- onIcon: Icon to display when on is true.
- offIcon: Icon to display when on is false.
- disabled: To disable the button.
- ariaLabelOn: aria-label of the button when on is true. If set, ariaLabelOff must also be set.
- ariaLabelOff: aria-label of the button when on is false. If set, ariaLabelOn must also be set.

#### Test Steps

- [x] `npm run storybook`
- [x] Navigate to Icon Button Toggle component in the side nav
- [x] Test the component

#### General Tests for Every PR

- [x] `npm run start` still works.
- [x] `npm run lint` passes.
- [x] `npm run stylelint` passes.
- [x] `npm test` passes and code coverage is not lower.
- [x] `npm run build` still works.

##### Screenshots
icon-button-toggle

### CODE REVIEW

File: libs/components/src/icon-button-toggle/icon-button-toggle.scss [hunk 1]
```diff
@@ -0,0 +1,13 @@
+:host {
+  background-color: var(--cv-theme-primary-8);
+  border-radius: 50%;
+  color: var(--cv-theme-primary);
```
[the remark this answers is not in the document]
Comment (vicalcantFrontEnd) [reply libs/components/src/icon-button-toggle/icon-button-toggle.scss:4]: looks good!

**Review by vicalcantFrontEnd — APPROVED** [review]

File: libs/components/src/icon-button-toggle/icon-button-toggle.scss [hunk 2]
```diff
@@ -0,0 +1,18 @@
+:host {
+  border-radius: 50%;
+  color: var(--cv-theme-on-surface-variant);
+
+  --mdc-ripple-color: var(--cv-theme-primary);
```
Comment (owilliams320) [inline libs/components/src/icon-button-toggle/icon-button-toggle.scss:5]: lets move this ripple color to only the "on" state as well.

**Review by owilliams320 — CHANGES_REQUESTED** [review]

File: libs/components/src/icon-button-toggle/icon-button-toggle.scss (hunk 2 above)
Comment (bsahitya) [reply libs/components/src/icon-button-toggle/icon-button-toggle.scss]: Consulted Adam and made changes to have the default grey ripple in the off state.

**Review by owilliams320 — APPROVED** [review]

**Pull request merged**